ssg/build_yaml.py

```diff
@@ -1563,13 +1565,28 @@ def export_ocil_to_xml(self):
         questionnaires = ET.SubElement(root, "{%s}questionnaires" % ocil_namespace)
         test_actions = ET.SubElement(root, "{%s}test_actions" % ocil_namespace)
         questions = ET.SubElement(root, "{%s}questions" % ocil_namespace)
-        for rule in self.rules.values():
+        return root, questionnaires, test_actions, questions
```

Comment:
We should return just root because the other items can be easily accessed trough root.